bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/FPValue.java

```diff
@@ -66,7 +67,7 @@ public void accept(T t) {
     }
 
     @Override
-    public String stringValue() {
+    public String stringValue(Strand strand) {
         return Optional.ofNullable(getType()).map(BType::toString).filter(str -> str.startsWith("function")).orElse(
```

Comment:
Something is wired in this line. and Possible NPE if type is null.

Reply:
fixed